onscale_client/api/util.py

```diff
@@ -9,16 +9,14 @@
 
 def wait_for_blob(blob_type: str,
                   object_id: str,
-                  object_type: str,
                   timeout_secs: float = None):
     """ Wait for a blob to exist of a specific type polling with backoff
     """
 
     def poll():
         """ Inner poll function to be retried """
         response = rest_api.blob_list_object(blob_type=blob_type,
-                                             object_id=object_id,
-                                             object_type=object_type)
+                                             object_id=object_id)
```

Comment:
The datamodel does contain the `object_type` field but this is the REST call:

![image](https://user-images.githubusercontent.com/61472294/206691699-822be1bb-2792-402c-9442-4b84a938335f.png)


Once you know the `object_id` it does not make sense to tell the server the object type, since each object id already knows which type it is. Do you want me to go back and add the additional field even though it is not used by the server? I don't like passing more arguments than needed.

Reply:
no, that makes sense if it is not used